src/main/java/com/google/api/generator/spring/composer/comment/CommentComposer.java

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 Google LLC
```

Comment:
Would importing `package com.google.api.generator.gapic.composer.comment.CommentComposer` work instead of adding this file?

Reply:
Removed class and put relevant used properties in AutoConfigCommentComposer/PropertiesCommentComposer